src/lib/blockchain.js

```diff
@@ -78,7 +79,7 @@ function getBalance(account) {
     });
 }
 
-export const getBlocks = (maxBlockCount = 3) => {
+export const getBlocks = (maxBlockCount) => {
```

Comment:
This breaks existing functionality.

This function is exported.
Your change means that wherever we now import it we no longer have the default value. Only when using it from getTransactions will it work